src/util/sss_tc_utf8.c

```diff
@@ -22,37 +22,21 @@
 #include "util/util.h"
 #include "util/sss_utf8.h"
 
-char *
-sss_tc_utf8_str_tolower(TALLOC_CTX *mem_ctx, const char *s)
-{
-    size_t nlen;
-    uint8_t *ret;
-
-    ret = sss_tc_utf8_tolower(mem_ctx, (const uint8_t *) s, strlen(s), &nlen);
-    if (!ret) return NULL;
+/* from sss_utf8.c */
+void sss_utf8_free(char *ptr);
+char *sss_utf8_tolower(const char *s);
```

Comment:
Hi,

I've you want to "hide" those functions, wouldn't it be better make move them into this file and make them static? You can do this in a second patch so that they changes you did to the implementation are easy to follow in this patch and then they are moved here with a second patch?

bye,
Sumit

Reply:
Well, this is what I meant under `"form" of those patches might be a topic to discuss` :)

Ok, moved.